### ISSUE TITLE
[BUG]: Custom external browser setting does not work

### ISSUE THREAD

**Issue by richardstevenhack** [martinrotter/rssguard#1303]
### Brief description of the issue

Yesterday I ran RSSGuard and was able to right click message links, open in external browser and the message would open in Firefox. Today it no longer does that.

I did a reinstall of RSSGuard 4.6.3 using the flatpak the other day. I don't know if that is related.

When I right click on a message and select open in external browser now, it gives me this pop-up:
`RSS Guard was unable to launch your web browser with the given URL, you need to open the below website URL in your web browser manually.`

At first I noticed that the default external browser was not set in Settings, so I added that. I am executing /usr/lib64/firefox/firefox as the executable. Running that from the command line runs the browser fine. Adding it to the external browser dialog in Settings inserts this command: /run/user/1000/doc/6b059135/firefox  Running that command from the command line gives me this error message: 

`rhack@localhost:/Data2/Work> /run/user/1000/doc/6b059135/firefox
Couldn't load XPCOM.
`

### How to reproduce the bug?

1) Retrieve feeds.
2) Right click on message
3) Select "Open in external browser".

### What was the expected result?

The expected result is the message link is opened in the default system browser.

### What actually happened?

I get this popup:
`RSS Guard was unable to launch your web browser with the given URL, you need to open the below website URL in your web browser manually.`

### Debug log

time="     0.120" type="debug" -> core: Initializing settings in '/home/rhack/.var/app/io.github.martinrotter.rssguard/config/RSS Guard 4/config/config.ini' (non-portable way).
time="     0.120" type="debug" -> core: Starting to load active localization. Desired localization is 'en_US'.
time="     0.121" type="debug" -> core: Application localization 'en_US' loaded successfully, specifically sublocalization 'en_US' was loaded.
time="     0.121" type="warning" -> core: Qt localization 'en_US' WAS NOT loaded successfully.
time="     0.168" type="debug" -> database: File-based SQLite database connection 'DatabaseFactory' to file '/home/rhack/.var/app/io.github.martinrotter.rssguard/config/RSS Guard 4/database/database.db' seems to be established.
time="     0.168" type="debug" -> database: File-based SQLite database has version '7'.
time="     0.168" type="debug" -> gui: Available icon theme paths: (/app/share/icons, /usr/share/icons, /usr/share/runtime/share/icons, /run/host/share/icons, :/icons, :/graphics, /home/rhack/.var/app/io.github.martinrotter.rssguard/config/RSS Guard 4/icons, /app/bin/icons)
time="     0.169" type="debug" -> gui: Installed icon themes are: '', 'Breeze_Snow', 'breeze', 'breeze-dark', 'breeze_cursors', 'hicolor', 'Adwaita', 'Breeze', 'Breeze Dark', 'Faenza', 'Numix'
time="     0.169" type="debug" -> gui: Loading icon theme 'Breeze'.
time="     0.169" type="debug" -> gui: Found path of base skin: ':/skins/nudus-base'.
time="     0.169" type="debug" -> gui: Base file ':/skins/nudus-base/html_wrapper.html' exists, using it for the skin.
time="     0.169" type="debug" -> gui: Local file ':/skins/nudus-light/html_style.css' exists, using it for the skin.
time="     0.169" type="debug" -> gui: Base file ':/skins/nudus-base/html_enclosure_image.html' exists, using it for the skin.
time="     0.169" type="debug" -> gui: Base file ':/skins/nudus-base/html_single_message.html' exists, using it for the skin.
time="     0.169" type="debug" -> gui: Base file ':/skins/nudus-base/html_enclosure_every.html' exists, using it for the skin.
time="     0.169" type="debug" -> gui: Local file ':/skins/nudus-light/qt_style.qss' exists, using it for the skin.
time="     0.169" type="debug" -> gui: Base file ':/skins/nudus-base/html_adblocked.html' exists, using it for the skin.
time="     0.169" type="debug" -> gui: Setting style: 'Fusion'.
time="     0.170" type="debug" -> gui: Skin 'nudus-light' loaded. Lite: 'false'.
time="     0.170" type="warning" -> core: User CSS style was not provided in file '/home/rhack/.var/app/io.github.martinrotter.rssguard/config/RSS Guard 4/web/user-styles.css'.
time="     0.170" type="debug" -> network: Persistent web data storage path: '/home/rhack/.var/app/io.github.martinrotter.rssguard/config/RSS Guard 4/web/storage'.
time="     0.170" type="debug" -> network: Disabling application-wide proxy completely.
time="     0.170" type="debug" -> core: SQLite version: '3.42.0'.
time="     0.171" type="debug" -> core: OpenSSL version: 'OpenSSL 3.1.5 30 Jan 2024'.
time="     0.171" type="debug" -> core: OpenSSL supported: 'true'.
time="     0.171" type="debug" -> core: Global thread pool has 32 threads.
time="     0.171" type="debug" -> core: Setting locale for LC_NUMERIC to C as libmpv requires it.
time="     0.171" type="debug" -> core: Instantiated class  'Application'.
time="     0.172" type="debug" -> database: SQLite database connection 'MessagesModel' to file '/home/rhack/.var/app/io.github.martinrotter.rssguard/config/RSS Guard 4/database/database.db' seems to be established.
time="     0.173" type="debug" -> message-model: Repopulated model, SQL statement is now:
 'SELECT Messages.id, Messages.is_read, Messages.is_important, Messages.is_deleted, Messages.is_pdeleted, Messages.feed, Messages.title, Messages.url, Messages.author, Messages.date_created, Messages.contents, Messages.enclosures, Messages.score, Messages.account_id, Messages.custom_id, Messages.custom_hash, Feeds.title, Feeds.is_rtl, CASE WHEN LENGTH(Messages.enclosures) > 10 THEN 'true' ELSE 'false' END AS has_enclosures, (SELECT GROUP_CONCAT(Labels.name) FROM Labels WHERE Messages.labels LIKE "%." || Labels.custom_id || ".%") as msg_labels, Messages.labels FROM Messages LEFT JOIN Feeds ON Messages.feed = Feeds.custom_id AND Messages.account_id = Feeds.account_id WHERE 0 > 1;'.
time="     0.173" type="debug" -> core: Auto-download timer started with interval 10000 ms.
time="     0.173" type="debug" -> core: Creating FeedDownloader singleton.
time="     0.182" type="debug" -> gui: Creating main application form in thread: '0x7f5d63b773c0'.
time="     0.185" type="debug" -> gui: Current row changed - proxy 'QModelIndex(-1,-1,0x0,QObject(0x0))', source 'QModelIndex(-1,-1,0x0,QObject(0x0))'.
time="     0.189" type="debug" -> network: Settings of BaseNetworkAccessManager loaded.
time="     0.228" type="debug" -> gui: Creating tray icon menu.
time="     0.252" type="debug" -> CTRL is NOT pressed while sorting articles - sorting with standard mode.
time="     0.252" type="debug" -> CTRL is NOT pressed while sorting articles - sorting with standard mode.
time="     0.253" type="debug" -> message-model: Repopulated model, SQL statement is now:
 'SELECT Messages.id, Messages.is_read, Messages.is_important, Messages.is_deleted, Messages.is_pdeleted, Messages.feed, Messages.title, Messages.url, Messages.author, Messages.date_created, Messages.contents, Messages.enclosures, Messages.score, Messages.account_id, Messages.custom_id, Messages.custom_hash, Feeds.title, Feeds.is_rtl, CASE WHEN LENGTH(Messages.enclosures) > 10 THEN 'true' ELSE 'false' END AS has_enclosures, (SELECT GROUP_CONCAT(Labels.name) FROM Labels WHERE Messages.labels LIKE "%." || Labels.custom_id || ".%") as msg_labels, Messages.labels FROM Messages LEFT JOIN Feeds ON Messages.feed = Feeds.custom_id AND Messages.account_id = Feeds.account_id WHERE 0 > 1 ORDER BY Messages.is_read ASC, Messages.date_created DESC;'.
time="     0.254" type="debug" -> core: Showing the main window when the application is starting.
time="     0.256" type="debug" -> database: SQLite database connection 'FeedReader' to file '/home/rhack/.var/app/io.github.martinrotter.rssguard/config/RSS Guard 4/database/database.db' seems to be established.
time="     0.256" type="debug" -> database: SQLite database connection 'FeedlyEntryPoint' to file '/home/rhack/.var/app/io.github.martinrotter.rssguard/config/RSS Guard 4/database/database.db' seems to be established.
time="     0.256" type="debug" -> database: SQLite database connection 'GmailEntryPoint' to file '/home/rhack/.var/app/io.github.martinrotter.rssguard/config/RSS Guard 4/database/database.db' seems to be established.
time="     0.256" type="debug" -> database: SQLite database connection 'GreaderEntryPoint' to file '/home/rhack/.var/app/io.github.martinrotter.rssguard/config/RSS Guard 4/database/database.db' seems to be established.
time="     0.256" type="debug" -> database: SQLite database connection 'OwnCloudServiceEntryPoint' to file '/home/rhack/.var/app/io.github.martinrotter.rssguard/config/RSS Guard 4/database/database.db' seems to be established.
time="     0.256" type="debug" -> database: SQLite database connection 'StandardServiceEntryPoint' to file '/home/rhack/.var/app/io.github.martinrotter.rssguard/config/RSS Guard 4/database/database.db' seems to be established.
time="     0.257" type="debug" -> database: SQLite database connection 'StandardServiceRoot' to file '/home/rhack/.var/app/io.github.martinrotter.rssguard/config/RSS Guard 4/database/database.db' seems to be established.
time="     0.257" type="debug" -> core: Custom ID of feed when loading from DB is '10'.
time="     0.257" type="debug" -> core: Custom ID of feed when loading from DB is '11'.
time="     0.257" type="debug" -> core: Custom ID of feed when loading from DB is '12'.
time="     0.257" type="debug" -> core: Custom ID of feed when loading from DB is '13'.
time="     0.257" type="debug" -> core: Custom ID of feed when loading from DB is '14'.
time="     0.257" type="debug" -> core: Custom ID of feed when loading from DB is '15'.
time="     0.257" type="debug" -> core: Custom ID of feed when loading from DB is '16'.
time="     0.258" type="debug" -> core: Custom ID of feed when loading from DB is '17'.
time="     0.258" type="debug" -> core: Custom ID of feed when loading from DB is '18'.
time="     0.258" type="debug" -> core: Custom ID of feed when loading from DB is 'new'.
time="     0.258" type="debug" -> core: Custom ID of feed when loading from DB is '20'.
time="     0.258" type="debug" -> core: Custom ID of feed when loading from DB is '21'.
time="     0.258" type="debug" -> core: Custom ID of feed when loading from DB is '22'.
time="     0.258" type="debug" -> core: Custom ID of feed when loading from DB is '23'.
time="     0.258" type="debug" -> core: Custom ID of feed when loading from DB is '24'.
time="     0.258" type="debug" -> core: Custom ID of feed when loading from DB is '25'.
time="     0.258" type="debug" -> core: Custom ID of feed when loading from DB is '26'.
time="     0.258" type="debug" -> core: Custom ID of feed when loading from DB is '27'.
time="     0.258" type="debug" -> core: Custom ID of feed when loading from DB is '28'.
time="     0.259" type="debug" -> core: Custom ID of feed when loading from DB is '29'.
time="     0.259" type="debug" -> core: Custom ID of feed when loading from DB is '30'.
time="     0.259" type="debug" -> core: Custom ID of feed when loading from DB is '31'.
time="     0.259" type="debug" -> core: Custom ID of feed when loading from DB is '32'.
time="     0.259" type="debug" -> core: Custom ID of feed when loading from DB is '33'.
time="     0.259" type="debug" -> core: Custom ID of feed when loading from DB is '34'.
time="     0.259" type="debug" -> core: Custom ID of feed when loading from DB is '35'.
time="     0.259" type="debug" -> core: Custom ID of feed when loading from DB is '36'.
time="     0.259" type="debug" -> core: Custom ID of feed when loading from DB is '37'.
time="     0.259" type="debug" -> core: Custom ID of feed when loading from DB is '38'.
time="     0.259" type="debug" -> core: Custom ID of feed when loading from DB is '39'.
time="     0.259" type="debug" -> core: Custom ID of feed when loading from DB is '40'.
time="     0.260" type="debug" -> core: Custom ID of feed when loading from DB is '41'.
time="     0.260" type="debug" -> core: Custom ID of feed when loading from DB is '42'.
time="     0.260" type="debug" -> core: Custom ID of feed when loading from DB is '43'.
time="     0.260" type="debug" -> core: Custom ID of feed when loading from DB is '44'.
time="     0.260" type="debug" -> core: Custom ID of feed when loading from DB is '45'.
time="     0.260" type="debug" -> core: Custom ID of feed when loading from DB is '46'.
time="     0.260" type="debug" -> core: Custom ID of feed when loading from DB is '47'.
time="     0.260" type="debug" -> core: Custom ID of feed when loading from DB is '58'.
time="     0.260" type="debug" -> core: Custom ID of feed when loading from DB is '59'.
time="     0.260" type="debug" -> core: Custom ID of feed when loading from DB is '60'.
time="     0.260" type="debug" -> core: Custom ID of feed when loading from DB is '61'.
time="     0.260" type="debug" -> core: Custom ID of feed when loading from DB is '62'.
time="     0.260" type="debug" -> core: Custom ID of feed when loading from DB is '63'.
time="     0.260" type="debug" -> core: Custom ID of feed when loading from DB is '64'.
time="     0.261" type="debug" -> core: Custom ID of feed when loading from DB is '65'.
time="     0.261" type="debug" -> core: Custom ID of feed when loading from DB is '66'.
time="     0.261" type="debug" -> core: Custom ID of feed when loading from DB is '67'.
time="     0.261" type="debug" -> core: Custom ID of feed when loading from DB is '68'.
time="     0.261" type="debug" -> core: Custom ID of feed when loading from DB is '69'.
time="     0.261" type="debug" -> core: Custom ID of feed when loading from DB is '70'.
time="     0.261" type="debug" -> core: Custom ID of feed when loading from DB is '71'.
time="     0.261" type="debug" -> core: Custom ID of feed when loading from DB is '72'.
time="     0.261" type="debug" -> core: Custom ID of feed when loading from DB is '73'.
time="     0.261" type="debug" -> core: Custom ID of feed when loading from DB is '74'.
time="     0.262" type="debug" -> core: Custom ID of feed when loading from DB is '75'.
time="     0.262" type="debug" -> core: Custom ID of feed when loading from DB is '76'.
time="     0.262" type="debug" -> core: Custom ID of feed when loading from DB is '77'.
time="     0.262" type="debug" -> core: Custom ID of feed when loading from DB is '78'.
time="     0.262" type="debug" -> core: Custom ID of feed when loading from DB is '79'.
time="     0.262" type="debug" -> core: Custom ID of feed when loading from DB is '80'.
time="     0.262" type="debug" -> core: Custom ID of feed when loading from DB is '81'.
time="     0.262" type="debug" -> core: Custom ID of feed when loading from DB is '82'.
time="     0.262" type="debug" -> core: Custom ID of feed when loading from DB is '83'.
time="     0.262" type="debug" -> core: Custom ID of feed when loading from DB is '84'.
time="     0.262" type="debug" -> core: Custom ID of feed when loading from DB is '85'.
time="     0.262" type="debug" -> core: Custom ID of feed when loading from DB is '86'.
time="     0.263" type="debug" -> core: Custom ID of feed when loading from DB is '87'.
time="     0.263" type="debug" -> core: Custom ID of feed when loading from DB is '88'.
time="     0.263" type="debug" -> core: Custom ID of feed when loading from DB is '89'.
time="     0.263" type="debug" -> core: Custom ID of feed when loading from DB is '90'.
time="     0.263" type="debug" -> core: Custom ID of feed when loading from DB is '91'.
time="     0.263" type="debug" -> core: Custom ID of feed when loading from DB is '92'.
time="     0.263" type="debug" -> core: Custom ID of feed when loading from DB is '93'.
time="     0.263" type="debug" -> core: Custom ID of feed when loading from DB is '94'.
time="     0.263" type="debug" -> core: Custom ID of feed when loading from DB is '95'.
time="     0.263" type="debug" -> core: Custom ID of feed when loading from DB is '96'.
time="     0.264" type="debug" -> core: Custom ID of feed when loading from DB is '97'.
time="     0.264" type="debug" -> core: Custom ID of feed when loading from DB is '99'.
time="     0.264" type="debug" -> core: Custom ID of feed when loading from DB is '100'.
time="     0.264" type="debug" -> core: Custom ID of feed when loading from DB is '101'.
time="     0.264" type="debug" -> core: Custom ID of feed when loading from DB is '102'.
time="     0.264" type="debug" -> core: Custom ID of feed when loading from DB is '103'.
time="     0.264" type="debug" -> core: Custom ID of feed when loading from DB is '104'.
time="     0.264" type="debug" -> core: Custom ID of feed when loading from DB is '105'.
time="     0.264" type="warning" -> core: Feed 'new' is loose, skipping it.
time="     0.264" type="debug" -> database: SQLite database connection 'RecycleBin' to file '/home/rhack/.var/app/io.github.martinrotter.rssguard/config/RSS Guard 4/database/database.db' seems to be established.
time="     0.265" type="debug" -> database: SQLite database connection 'ImportantNode' to file '/home/rhack/.var/app/io.github.martinrotter.rssguard/config/RSS Guard 4/database/database.db' seems to be established.
time="     0.265" type="debug" -> database: SQLite database connection 'UnreadNode' to file '/home/rhack/.var/app/io.github.martinrotter.rssguard/config/RSS Guard 4/database/database.db' seems to be established.
time="     0.265" type="debug" -> database: SQLite database connection 'LabelsNode' to file '/home/rhack/.var/app/io.github.martinrotter.rssguard/config/RSS Guard 4/database/database.db' seems to be established.
time="     0.265" type="debug" -> database: SQLite connection 'StandardServiceRoot' is already active.
time="     0.265" type="debug" -> database: SQLite database connection 'StandardServiceRoot' to file '/home/rhack/.var/app/io.github.martinrotter.rssguard/config/RSS Guard 4/database/database.db' seems to be established.
time="     0.266" type="debug" -> database: SQLite database connection 'TtRssServiceEntryPoint' to file '/home/rhack/.var/app/io.github.martinrotter.rssguard/config/RSS Guard 4/database/database.db' seems to be established.
time="     0.266" type="debug" -> gui: User wants to have tray icon.
time="     0.266" type="warning" -> gui: Showing tray icon with little delay.
time="     0.266" type="debug" -> gui: Feed list item expanded - rhack (RSS/ATOM/JSON)
time="     0.266" type="debug" -> gui: Feed list item expanded - Original Feeds
time="     0.266" type="debug" -> gui: Feed list item expanded - Imported Feeds
time="     0.266" type="debug" -> core: No execution message received from other app instances.
time="     0.330" type="debug" -> gui: Hovered link: 'QUrl()'.
time="     3.184" type="warning" -> gui: Tray icon is available, showing now.
time="     3.187" type="debug" -> gui: Creating SystemTrayIcon instance.
time="     3.187" type="debug" -> gui: Showing tray icon immediately.
time="     3.195" type="debug" -> gui: Tray icon displayed.
time="     3.322" type="debug" -> CTRL is NOT pressed while sorting articles - sorting with standard mode.
time="     3.322" type="debug" -> core: Displaying messages from feeds IDs: ''25', '26', '27', '28', '29', '30', '31', '32', '33', '34', '35', '36', '37', '38', '39', '40', '41', '42', '43', '44', '45', '46', '47'' and URLs: 'http://feeds.feedburner.com/DanchoDanchevOnSecurityAndNewMedia, http://windowsir.blogspot.com/feeds/posts/default, http://feeds.feedburner.com/KrebsOnSecurity, http://feeds.feedburner.com/PentestTools, https://www.darknet.org.uk/feed/, https://www.kalilinux.in/feeds/posts/default, https://isc.sans.edu/rssfeed.xml, http://www.social-engineer.org/feed/, http://feeds.feedburner.com/TheHackersNews?format=xml, https://www.schneier.com/blog/atom.xml, http://feeds.feedburner.com/darknethackers, http://threatpost.com/feed, https://www.bleepingcomputer.com/feed/, http://packetstormsecurity.org/headlines.xml, http://taosecurity.blogspot.com/atom.xml, http://www.techrepublic.com/rssfeeds/topic/security/?feedType=rssfeeds, https://www.exploitone.com/feed/, https://www.bleepingcomputer.com/tutorials/feed/, https://follow.it/haxf4rall1, https://www.securedyou.com/feed/, http://securityaffairs.co/wordpress/feed, http://www.forensickb.com/feeds/posts/default, https://www.wilderssecurity.com/forums/-/index.rss'.
time="     3.323" type="debug" -> message-model: Repopulated model, SQL statement is now:
 'SELECT Messages.id, Messages.is_read, Messages.is_important, Messages.is_deleted, Messages.is_pdeleted, Messages.feed, Messages.title, Messages.url, Messages.author, Messages.date_created, Messages.contents, Messages.enclosures, Messages.score, Messages.account_id, Messages.custom_id, Messages.custom_hash, Feeds.title, Feeds.is_rtl, CASE WHEN LENGTH(Messages.enclosures) > 10 THEN 'true' ELSE 'false' END AS has_enclosures, (SELECT GROUP_CONCAT(Labels.name) FROM Labels WHERE Messages.labels LIKE "%." || Labels.custom_id || ".%") as msg_labels, Messages.labels FROM Messages LEFT JOIN Feeds ON Messages.feed = Feeds.custom_id AND Messages.account_id = Feeds.account_id WHERE Feeds.custom_id IN ('25', '26', '27', '28', '29', '30', '31', '32', '33', '34', '35', '36', '37', '38', '39', '40', '41', '42', '43', '44', '45', '46', '47') AND Messages.is_deleted = 0 AND Messages.is_pdeleted = 0 AND Messages.account_id = 1 ORDER BY Messages.is_read ASC, Messages.date_created DESC;'.
time="    17.682" type="debug" -> gui: Message list got focus with reason 'Qt::MouseFocusReason'.
time="    17.682" type="debug" -> gui: Current row changed - proxy 'QModelIndex(46,6,0x55b218d9bad0,MessagesProxyModel(0x55b218d4cd10, name = MessagesProxyModel))', source 'QModelIndex(46,6,0x0,MessagesModel(0x55b218d222f0))'.
time="    17.781" type="debug" -> gui: Hovered link: 'QUrl()'.
time="    19.896" type="debug" -> gui: Message list got focus with reason 'Qt::PopupFocusReason'.
time="    19.897" type="debug" -> network: We are trying to open URL 'https://www.techrepublic.com/article/best-free-password-managers/'.
time="    19.897" type="debug" -> network: Arguments for external browser: '"https://www.techrepublic.com/article/best-free-password-managers/"'.
time="    19.909" type="debug" -> network: External web browser call failed.
time="    22.178" type="debug" -> database: SQLite database connection 'StandardFeed' to file '/home/rhack/.var/app/io.github.martinrotter.rssguard/config/RSS Guard 4/database/database.db' seems to be established.
time="    22.178" type="debug" -> database: SQLite connection 'UnreadNode' is already active.
time="    22.179" type="debug" -> database: SQLite database connection 'UnreadNode' to file '/home/rhack/.var/app/io.github.martinrotter.rssguard/config/RSS Guard 4/database/database.db' seems to be established.
time="    22.179" type="debug" -> database: SQLite connection 'StandardServiceRoot' is already active.
time="    22.179" type="debug" -> database: SQLite database connection 'StandardServiceRoot' to file '/home/rhack/.var/app/io.github.martinrotter.rssguard/config/RSS Guard 4/database/database.db' seems to be established.
time="    22.179" type="debug" -> feed-model: There is request to reload feed model, reloading the 2 items individually.
time="    22.192" type="debug" -> gui: Message list got focus with reason 'Qt::ActiveWindowFocusReason'.
time="    25.194" type="debug" -> gui: Message list got focus with reason 'Qt::PopupFocusReason'.
time="    25.224" type="debug" -> core: Cleaning up resources and saving application state.
time="    25.224" type="debug" -> core: Close lock was obtained safely.
time="    25.226" type="debug" -> core: Stopping StandardServiceRoot instance.
time="    25.226" type="debug" -> feed-downloader: Destroying FeedDownloader instance.
time="    25.227" type="debug" -> gui: Destroying FormMain instance.
time="    25.228" type="debug" -> gui: Destroying TabWidget instance.
time="    25.228" type="debug" -> gui: Destroying FeedMessageViewer instance.
time="    25.229" type="debug" -> gui: Destroying BaseToolBar instance.
time="    25.229" type="debug" -> network: Destroying Downloader instance.
time="    25.229" type="debug" -> network: Destroying SilentNetworkAccessManager instance.
time="    25.233" type="debug" -> gui: Destroying MessagesView instance.
time="    25.260" type="debug" -> gui: Destroying BaseToolBar instance.
time="    25.260" type="debug" -> gui: Destroying FeedsView instance.
time="    25.260" type="debug" -> gui: Destroying TabBar instance.
time="    25.261" type="debug" -> gui: Destroying StatusBar instance.
time="    25.261" type="debug" -> gui: Destroying SystemTrayIcon instance.
time="    25.268" type="debug" -> core: Destroying Application instance.
time="    25.268" type="debug" -> core: Destroying Mutex instance.
time="    25.325" type="debug" -> gui: Destroying IconFactory instance.
time="    25.326" type="debug" -> core: Destroying FeedReader instance.
time="    25.326" type="debug" -> feed-model: Destroying FeedsModel instance.
time="    25.326" type="debug" -> feed-model: Destroying FeedsProxyModel instance
time="    25.326" type="debug" -> message-model: Destroying MessagesModel instance.
time="    25.326" type="debug" -> message-model: Destroying MessagesProxyModel instance.

### Operating system and version

* OS:  openSUSE Tumbleweed 20240220
KDE Plasma Version: 5.27.10
KDE Frameworks Version: 5.115.0
Qt Version: 5.15.12
Kernel Version: 6.7.5-1-default (64-bit)
Graphics Platform: X11
Processors: 32 × AMD Ryzen 9 5950X 16-Core Processor
Memory: 62.7 GiB of RAM
Graphics Processor: AMD Radeon RX 550 Series
* RSS Guard version: Version: 4.6.3 (built on Linux/x86_64)
Revision: 
Build date: 1/22/24 7:12 PM
Qt: 5.15.10 (compiled against 5.15.10)

**Comment by martinrotter** [comment]
Flatpakked version of RSS Guard is sandboxed, meaning it cannot just use arbitrary files outside of allowed paths. This functionality is limited by that fact.

https://docs.flatpak.org/en/latest/sandbox-permissions.html

You can add permissions to flatpak apps.

https://askubuntu.com/questions/1086529/how-to-give-a-flatpak-app-access-to-a-directory

That should help you to run firefox directly from RSS Guard I guess.

**Comment by richardstevenhack** [comment]
I did give RSSGuard access to the entire file system. That's not the problem.

**Comment by martinrotter** [comment]
I see. I will have to test on my Linux machine, but I am really not flatpak core user...

**Comment by guihkx** [comment]
>I did give RSSGuard access to the entire file system. That's not the problem.

That won't do anything in terms of allowing a sandboxed Flatpak app to run the binaries in `/usr/bin`.

If you want to do it correctly, I've written a quick how-to here:

https://github.com/martinrotter/rssguard/issues/917#issuecomment-1517525689

TL;DR:

1\. Break the sandbox:

```
flatpak --user override --talk-name=org.freedesktop.Flatpak io.github.martinrotter.rssguard
```

2\. Restart RSS Guard

3\. Set your "web browser executable" to:

```
/usr/bin/flatpak-spawn
```

4\. Use this as your parameters (replacing `firefox` by your browser of choice):

```
--host firefox "%1"
```

@martinrotter Btw, feel free to mention me whenever it's a Flatpak issue.
